src/main/java/org/opentripplanner/visualizer/GraphVisualizer.java

```diff
@@ -625,17 +625,13 @@ private JComponent makePrefsPanel(){
         bikeCheckBox = new JCheckBox("bike");
         pane.add(bikeCheckBox);
 
-        // @flibbertigibbet:  replacing these to replace with test checkboxes for benches/toilets
-        /*
-        trainCheckBox = new JCheckBox("trainish");
-        pane.add(trainCheckBox);
-        ferryCheckBox = new JCheckBox("ferry");
-        pane.add(ferryCheckBox);
-        */
-        benchCheckBox = new JCheckBox("Prefer benches");
-        pane.add(benchCheckBox);
-        toiletCheckBox = new JCheckBox("Prefer toilets");
-        pane.add(toiletCheckBox);
+		//////////////////////////////
+        restCheckBox = new JCheckBox("Prefer resting spots");
+        pane.add(restCheckBox);
+        allowUnevenCheckBox = new JCheckBox("Allow uneven surfaces");
+		allowUnevenCheckBox.setSelected(true);
+        pane.add(allowUnevenCheckBox);
```

Comment:
I didn't see a checkbox for wheelchair, if it's trivial would be helpful to add one in there for that as well.